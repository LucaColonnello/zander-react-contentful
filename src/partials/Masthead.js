import React from 'react';
import { Link } from 'react-router-dom';
import Nav from './Nav';

const Masthead = () => {
  return (
    <header id="masthead" className="masthead" role="banner">
      <Link to="/" className="logo">
        <svg className="logo-icon"><use xlinkHref="#icon-zmLogo" /></svg>
        <span className="is-visuallyHidden">Zander Martineau</span>
      </Link>
      <Nav className="nav--masthead" />
    </header>
  );
};

export default Masthead;
