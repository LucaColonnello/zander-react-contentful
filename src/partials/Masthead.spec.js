import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Masthead from './Masthead'

test('Masthead snapshot test', () => {
  const component = shallow(
    <Masthead />
  )
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
