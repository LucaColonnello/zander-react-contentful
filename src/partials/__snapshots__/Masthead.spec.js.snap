// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Masthead snapshot test 1`] = `
<header
  className="masthead"
  id="masthead"
  role="banner"
>
  <Link
    className="logo"
    replace={false}
    to="/"
  >
    <svg
      className="logo-icon"
    >
      <use
        xlinkHref="#icon-zmLogo"
      />
    </svg>
    <span
      className="is-visuallyHidden"
    >
      Zander Martineau
    </span>
  </Link>
  <Nav
    className="nav--masthead"
  />
</header>
`;
